packages/components/src/__actions__/Button/v3/_docs/Button--api-specification.mdx

```diff
@@ -0,0 +1,160 @@
+
+<Canvas of={exampleStories.Playground} />
+
+## API
+
+This is built on top [React Aria's Button component](https://react-spectrum.adobe.com/react-aria/Button.html#props) and seeks to provide a flexible API with convenient abstractions that handle variants, sizes, pending states and icons.
+
+The following table showcases the essential props that enable the core functionality of `Button`.
+
+<Controls of={exampleStories.Playground} className="mb-64" include={["children", "className",  "size", "variant", "onPress", "icon", "iconPosition", "isFullWidth", "isDisabled", "isPending", "isPendingLabelHidden",  "pendingLabel"]} />
+
+### `onPress` vs `onClick`
+
+One key change to the `Button`&apos;s API is the shift from `onClick` to `onPress`. This will not require a change to the implementation of `onClick` logic, ie:
+
+```tsx
+<Button label="Submit" onClick={e => sumbit(e)}/>
+```
+
+Can safely be replaced with the following:
+
+```tsx
+<Button onPress={e => submit(e)}>
+  Submit
+</Button>
+```
+
+React Aria&apos;s implementation of `onPress` provides better support for consistent touch events across device types, such mobile, desktop and tablet. You can read more about the development and reason behind this pattern [here](https://react-spectrum.adobe.com/blog/building-a-button-part-1.html#touch-interactions).
+
+
+### Button content and children
+
+Labels and any button content can be passed into the `Button` as `children`. Content wrapped by the `Button` will be spaced using `flex-gap` relative to the `size` prop. This enables consistent spacing in `RTL` layouts.
+
+```tsx
+<Button variant="secondary" onPress={e => submit(e)}>
+  Label
+</Button>
+```
+
+While in most cases, `children` will be a `ReactNode`, `Button` also accepts a render function with React Aria's `ButtonRenderProps`. This allows for more advanced styling and rendering options by hooking into React Aria's internal button state. You can read more about this [here](https://react-spectrum.adobe.com/react-aria/Button.html#styling).
+
+
+### Icons and positioning
+
+The `icon` property abstracts the need to handle positioning and sizing logic for icons within the `Button`. When paired with the [Icon component](/docs/illustrations-icon-icon-future-api-specification--docs), this will scale optical size and stroke width `size` prop.
+
+<Canvas of={exampleStories.ButtonWithIcon} />
+
+Set the position of the icon using the `iconPosition` prop. This will ensure content is flipped in `RTL` layouts.
+
+<Canvas of={exampleStories.ButtonWithIconEnd} />
+
+Note that icons will need the [shouldMirrorInRTL](/docs/illustrations-icon-icon-future-api-specification--docs#mirror-in-rtl) prop set to `true` when mirroring is required.
+
+
+### Icon only buttons
+
+The recommended pattern to achieve an icon only button (aka: `IconButton`) is to pass an icon in as props, then to use a [VisuallyHidden](/docs/components-visually-hidden--docs) component as `children`.
```

Comment:
I'd change the "aka: IconButton" to either "previously" or remove it since it sounds like we actually still have this component and it is a current alternative option.

Reply:
Fair point 👍